test/Sentry.Extensions.Logging.Tests/LoggingTests.cs

```diff
@@ -14,6 +14,11 @@ public class LoggingTests
     [InlineData(LogLevel.Trace)]
     public void Log_CapturesEvent(LogLevel logLevel)
     {
+#if __IOS__
+        Skip.If(true, "Flaky on iOS");
+#endif
+        Skip.If(TestEnvironment.IsGitHubActions, "Flaky on CI");
```

Comment:
This is flaky in CI and on iOS? should we get rid of this test?

Reply:
Yeah the stuff I surrounded in directives is flaky on my local dev machine as well as CI for that target platform. So in this case it's always flaky on iOS, but only flaky in CI on Android (runs reliably on Android on my local dev machine)... worth pointing out that they're only flaky on Android targeting API levels 28+ (reliable when targeting API level 27). 

There are some comments in the PR description about a potential way to resolve on Android, so it might be worth leaving the tests in here (disabled on CI), in the hope that we could circle back to this.

Reply:
> There are some comments in the PR description about a potential way to resolve on Android, so it might be worth leaving the tests in here (disabled on CI), in the hope that we could circle back to this.

Worth adding the notes on a ticekt since whatever is left here will be forgotten (unless a ticket links back to it)

Reply:
Done in https://github.com/getsentry/sentry-dotnet/issues/3649